src/main/java/ByeCommand.java

```diff
@@ -0,0 +1,10 @@
+/**
+ * Represents a "bye" command given to TaskList.
+ */
+public class ByeCommand extends Command {
+
+    public ByeCommand() {
+        super();
```

Comment:
I personally think `super()` is quite hard to understand if I haven't followed the implementation for class `Command`. Not sure if we can improve on this issue.